### PR TITLE
feat: implement UI only

### PR DESCRIPTION
**Pull request by doinel1a**
Implement wallet connection in nav.

### CODE REVIEW

**Comment by doinel1a** [comment]
Implement Header, Templates & Prompt sections in Home page.

**Comment by doinel1a** [comment]
Implement 404 page.

File: front-end/src/pages/four-ou-four.tsx
Comment (urataps) [inline front-end/src/pages/four-ou-four.tsx:1]: Maybe let's use 404 naming, this four-ou-four looks funky, i.e. 404.tsx and `function 404Page()`

File: front-end/src/assets/fonts/PlusJakartaSans-Bold.woff
Comment (urataps) [inline front-end/src/assets/fonts/PlusJakartaSans-Bold.woff:1]: Best font ever

**Comment by doinel1a** [comment]
Implement Code Viewer w/ copy & download buttons.

File: front-end/src/pages/four-ou-four.tsx
Comment (doinel1a) [reply front-end/src/pages/four-ou-four.tsx:1]: I would, but in all programming languages - or almost - variables, methods & file names cannot start with a number.

Comment (urataps) [reply front-end/src/pages/four-ou-four.tsx:1]: or NotFound is also fine

**Review by urataps — APPROVED** [review]
lgtm

**Comment by doinel1a** [comment]
Minor refactorings.

**Comment by doinel1a** [comment]
Minor refactorings.

**Pull request merged**